mesonbuild/mesonlib.py

```diff
@@ -356,6 +356,9 @@ def is_haiku() -> bool:
 def is_openbsd() -> bool:
     return platform.system().lower() == 'openbsd'
 
+def is_sunos() -> bool:
```

Comment:
I mentioned this before, but let's make this `is_solaris()` and use `solaris` instead of `sunos`.

You should also add solaris to the Operating system table in the docs.

Reply:
Done.